cmd/root.go

```diff
@@ -20,11 +22,14 @@ func NewRootCmd() *cobra.Command {
 		Short:        "REANA client for interacting with REANA server.",
 		Long:         "REANA client for interacting with REANA server.",
 		SilenceUsage: true,
+		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+			return runE(cmd)
+		},
 	}
 
 	cmd.SetOut(os.Stdout)
 
-	cmd.PersistentFlags().BoolP("loglevel", "l", false, "Sets log level [DEBUG|INFO|WARNING]")
+	cmd.PersistentFlags().StringP("loglevel", "l", "WARNING", "Sets log level [DEBUG|INFO|WARNING]")
```

Comment:
What do you think about using a struct to save this flag, just like we did for the other commands in https://github.com/reanahub/reana-client-go/pull/56?